pom.xml

```diff
@@ -185,7 +185,7 @@
                         <configuration>
                             <referenceVersion>2.22ea1</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
-                            <binaryCompatibilityPercentageRequired>99</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>94</binaryCompatibilityPercentageRequired>
```

Comment:
I think we should document every wilful API break in the POM using comments when we reduce the percentage required. Some API changes (like making an abstract class' constructor protected rather than public) are safe and others are not.

Reply:
We should just put 2.23ea0 there, after all, it's 2.23 time?